download-directory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -58,10 +65,10 @@ public static void main(String[] args) throws Exception {
 			releaseDir.mkdir();
 		}
 
-		String pathToSpeciesConfig = props.getProperty("speciesConfigPath");
+		String pathToSpeciesConfig = "src/main/resources/Species.json";
```

Comment:
Using `Paths.get("src/main/resources/Species.json").toString()` might be better just in case so the system takes care of the directory separators for you.

https://stackoverflow.com/questions/16919501/create-a-path-from-string-in-java7